am-lib/src/test/java/integration/uk/gov/hmcts/reform/amlib/GrantAccessIntegrationTest.java

```diff
@@ -65,4 +80,23 @@ void whenCreatingResourceForMultipleUsersShouldAppearInDatabase() {
         assertThat(databaseHelper.findExplicitPermissions(resourceId)).hasSize(2)
             .extracting(ExplicitAccessRecord::getAccessorId).containsOnly("y", "z");
     }
+
+    @Test
+    void whenCreatingResourceWithInvalidRelationshipShouldThrowPersistenceException() {
+        ExplicitAccessGrant nonExistingRole = ExplicitAccessGrant.builder()
```

Comment:
Why not to use `createGrant` method here. No much point in having 10+ lines of object initiation.

Reply:
👍